src/core/settings.c

```diff
@@ -119,6 +119,9 @@ QuicSettingsSetDefault(
     if (!Settings->IsSet.StatelessOperationExpirationMs) {
         Settings->StatelessOperationExpirationMs = QUIC_STATELESS_OPERATION_EXPIRATION_MS;
     }
+    if (!Settings->IsSet.CongestionControlAlgorithm) {
+        Settings->CongestionControlAlgorithm = QUIC_CONGESTION_CONTROL_ALGORITHM_CUBIC;
```

Comment:
We should have a separate define (in quicdef.h) for `QUIC_CONGESTION_CONTROL_ALGORITHM_DEFAULT` and default it to cubic.